documentation/src/main/docs/kafka/transactions.md

```diff
@@ -0,0 +1,92 @@
+# Kafka Transactions and Exactly-Once Processing
+
+!!!warning "Experimental"
+    Kafka Transactions is an experimental feature.
+
+[Kafka transactions](https://docs.google.com/document/d/11Jqy_GjUGtdXJK94XGsEIK7CP1SnQGdp2eF0wSw9ra8/edit) enable atomic writes to multiple Kafka topics and partitions.
```

Comment:
Are you sure about linking that document?

Reply:
Other than this document, I could not find any mention of the Kafka transactions on the official kafka.apache.org site.  We can include a link to one of the Confluent sites though.

Reply:
Let me ask around. Let's keep the google doc for now.